.circleci/config.yml

```diff
@@ -23,7 +23,6 @@ commands:
     parameters:
       subdirectory:
         type: string
-        default: ''
```

Comment:
for my own understanding, where do we get the subdirectory parameter? is that always given so there's no need for a default?

Reply:
All of the calls to `deploy-to-aws` specify a subdirectory. Be removing the default, I'm simply making it a required parameter since we no longer want to update the files at the top level of the folder